CHANGES.md

```diff
@@ -1,3 +1,19 @@
+### v1.2.3
+
+- Update to cmdliner 1.1.0 (@MisterDA #249)
+
+- Fix tests on Windows, avoid using Unix domain sockets (@talex5 #251)
+
+- Fix warning on Unit.ENOTCONN under macos (@talex5 #252)
```

Comment:
```suggestion
- Fix warning on `Unix.ENOTCONN` under macos (@talex5 #252)
```
BTW, are you generating the changes updates manually? I have a little script to do this that you might like...

Reply:
Yes, I'm doing this manually by browsing the diff between tags and master. 
Automation would be appreciated.